comixed-web/src/app/library/actions/comic-details.actions.ts

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
I just realized there's no reducer on this PR. Looking it over, I think we should move all of this feature behavior into the existing Library feature where it's already managing the currently displayed comic. Then you would want to add a "saving" flag to the state and have the ComicDetails component set the busy indicator based on saving as well as loading.